benchmarking.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    apt-get install unzip -y
</pre><p> </p><p>Can you add some explanations here for user who dont know what's going on here:</p><p> </p><ul><li>what data is used?</li><li>From where does it take the trained models? Who trained the models? &nbsp;what script we should use to train the model and export it?</li><li>what does this <code>rees46_ecom_dataset_small_for_ci.zip</code> file  include? and did you generate it? does it have the trained models in it as exported?</li></ul><p> </p><p>Thanks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/615/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Added a whole new notebook for documenting training! 🙂 Will continue to keep information as I go.</p>